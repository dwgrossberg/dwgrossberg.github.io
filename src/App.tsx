import { useState } from "react";
import reactLogo from "./assets/react.svg";
import "./App.css";

function App() {
  const [count, setCount] = useState(0);

  return (
    <div className="App">
      <h1>Daniel Grossberg</h1>
      <h2>Software Engineer</h2>
      <h4>coming soon...</h4>
    </div>
  );
}

export default App;
