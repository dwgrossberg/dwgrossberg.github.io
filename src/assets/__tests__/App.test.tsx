/**
 * @jest-environment jsdom
 */
import { describe, it, expect, afterEach } from "vitest";
import { render, screen, cleanup } from "@testing-library/react";
import App from "../../App";

describe("App component", () => {
  afterEach(cleanup);
  it("snapshot test", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
