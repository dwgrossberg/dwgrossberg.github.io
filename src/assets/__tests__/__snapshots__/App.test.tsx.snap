// Vitest Snapshot v1

exports[`App component > snapshot test 1`] = `
<div>
  <div
    class="App"
  >
    <h1>
      Daniel Grossberg
    </h1>
    <h2>
      Software Engineer
    </h2>
    <h4>
      coming soon...
    </h4>
  </div>
</div>
`;
